.travis.yml

```diff
@@ -0,0 +1,30 @@
+dist: xenial
+language: python
+python:
+  - "3.4"
+  - "3.5"
+  - "3.6"
+  - "3.7"
+env:
+  - DJANGO=1.11.*
+  - DJANGO=2.0.*
+  - DJANGO=2.1.*
+  - DJANGO=2.2.*
+install:
+  - pip install Django==$DJANGO django-macros
```

Comment:
I think this will test the version that's currently on pip, though I think it'd make more sense to test the version in master. Should this line be followed by `pip install .` instead?